src/components/BioMose.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import mose from '../../src/img/mose.png'
+
+const BioMose = () => {
+  return (
+    <div className="bg-slate-50">
+      <div className="px-6 md:px-16 lg:px-24 pt-8 md:pt-16">
+        <h2 className="text-5xl md:text-6xl mb-3 text-center">About</h2>
+        <hr className="mx-4 md:mx-20 bg-black" />
+        <p className="py-8 md:py-12 text-lg md:text-xl">
+          Owner, Mose Primus is a long-time resident of Eastside Detroit and is the president of the local neighborhood association. Through his volunteer and non-profit work, Mose works tirelessly to beautify his neighborhood, mitigate blight, and create a safe, welcoming space for all residents.
+        </p>
+      </div>
+      <img src={mose} alt="Mose Primus" className="w-full h-96 object-cover" />
```

Comment:
issue: On wider screens, his head gets cut off! If the Figma has this image as going across the whole screen, then get with the designer to ask what you should do in a case like this.

issue: Sometimes as developers we might be forced to use the name as the alt text, but here we can do better, since we know the person and the picture. Check in with @ajkarczewski to ask for suitable alt text, and please have a read through the [Alt Text As Poetry website](https://alt-text-as-poetry.net/) - it's one of my favourite resources for starting to learn how to write alt text well.